node/modules/storageminer.go

```diff
@@ -683,3 +683,9 @@ func NewOnlineBackupMgr(cfg *config.Boost) func(lc fx.Lifecycle, r lotus_repo.Lo
 		return backupmgr.NewBackupMgr(r, ds, db.DealsDBName, dealsDB)
 	}
 }
+
```

Comment:
Do we need this change for backup?

Reply:
It came because of merge conflict in file as both func() were in same line. Just resolved the conflict and 1 empty line between 2 funcs.

Reply:
Ok thanks, feel free to merge